src/main.cpp

```diff
@@ -59,7 +61,41 @@ void sokoban_initialize() {
             "######",
         },
     };
-    soko = {levels};
+    soko = {levels};*/
+    //std::vector <std::vector<std::string>> levels;
+    std::filesystem::path getLevelsDir = std::filesystem::directory_entry("dist/levels");
+    std::vector<std::filesystem::path> getMapPath; // save path into vector element
+
+    for (const auto& entry : std::filesystem::directory_iterator(getLevelsDir)) {
+        getMapPath.push_back(entry.path());
+    }
+
+    // removes every other instance of a file found in the directory, namely the .data file since they are every other in the directory, to leave only .txt files in the vector
```

Comment:
Confusing comment -- there should be an easier way with a single loop and no `getMapPath` vector.